src/components/Avatar/spec.js

```diff
@@ -0,0 +1,39 @@
+/* globals expect, describe, it */
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { mount } from 'enzyme'
+import { Avatar, Provider } from '../../../dist'
+
+describe('Avatar component', () => {
+  it('Avatar renders correctly', () => {
+    const component = renderer.create(
+      <Provider>
+        <Avatar />
+      </Provider>
+    )
+    let tree = component.toJSON()
+    expect(tree).toMatchSnapshot()
+  })
+
+  describe('firstName and lastName properties', () => {
+    it('should render the Avatar with letters TU', () => {
```

Comment:
```suggestion
    it('should render the Avatar with initials if src is not passed', () => {
```

We don't want to specify the assertion value as part of the test name, that's not what we are testing.